orc8r/gateway/go/services/magmad/status/poller.go

```diff
@@ -39,7 +39,7 @@ var (
 )
 
 func startServiceQuery(service string, queryMetrics bool, maxMetricsQueueSz int) error {
-	conn, err := service_registry.Get().GetConnection(strings.ToUpper(service), protos.ServiceType_SOUTHBOUND)
+	conn, err := service_registry.Get().GetConnection(strings.ToUpper(service), protos.ServiceType_PROTECTED)
```

Comment:
I think the orc8r magmad is a special case. Even though it's in the orc8r folder, it is actually supposed to be running on gateways, and should not use the protected port.

Reply:
Fixed.